dags/scrape_factory.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from airflow import DAG
+
+from dags.config import SCRAPING_DAGS
+from dags.constants import LA_METRO_DATABASE_URL, AIRFLOW_DIR_PATH, \
+    DAG_DESCRIPTIONS, START_DATE
+from operators.blackbox_docker_operator import BlackboxDockerOperator
+
+
+docker_base_environment = {
+    'DECRYPTED_SETTINGS': 'pupa_settings.py',
+    'DESTINATION_SETTINGS': 'pupa_settings.py',
+    'DATABASE_URL': LA_METRO_DATABASE_URL,  # For use by entrypoint
+    'LA_METRO_DATABASE_URL': LA_METRO_DATABASE_URL,  # For use in scraping scripts
+}
+
+for dag_name, dag_config in SCRAPING_DAGS.items():
+    for idx, interval in enumerate(dag_config['schedule_interval'], start=1):
+        dag_id = dag_name if len(dag_config['schedule_interval']) == 1 else '{0}_{1}'.format(dag_name, idx)
```

Comment:
This was a quick and dirty way to give DAGs unique names if we have multiple schedules. We might want to do something more description, e.g., parse the weekday from the cron expression.